data/json/vehicleparts/frames.json

```diff
@@ -81,16 +81,16 @@
     "broken_color": "brown",
     "difficulty": 0,
     "durability": 100,
-    "requirements": {
+    "requirements": [
       "install": {
```

Comment:
That explains it. Next commit should resolve that, as I accidentally synced after reverting. I should have removed the mistakes the hard way.